plexus-compilers/plexus-compiler-aspectj/src/main/java/org/codehaus/plexus/compiler/ajc/AspectJCompiler.java

```diff
@@ -25,9 +25,14 @@
 import java.net.URLClassLoader;
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
+
+import static org.aspectj.org.eclipse.jdt.internal.compiler.classfmt.ClassFileConstants.*;
```

Comment:
I'd prefer not to static import these as they're not familiar and common so readers will wonder where they came from.

Reply:
Sorry, but I use so many of them, it is just too wordy otherwise with all Java versions from 1.1 to 16. You would not see the forest because of the many trees anymore. And I expect other readers (who are always developers) to either understand what a static import is - or let their IDE help them when they jump to a definition.

Reply:
You can't rely on IDEs here. For instance, I am currently reading this code in my web browser, not an IDE.

Reply:
I said "or IDE". You don't need to rely on it in order to understand the code. I hope you don't mind that I am not going to cater to your personal preference of 19 static imports (I counted!), because you think it makes the code more readable. I simply disagree.

Reply:
Codehaus style is explicit imports, consistency is key over personal preferences.

Reply:
Oh OK, I did not know that. Fine, I am going to change it.